services/data-record-service.js

```diff
@@ -41,12 +41,23 @@ class DataRecordService {
     async validateMetadata(submissionID, types, scope) {
         isValidMetadata(types, scope);
         const isMetadata = types.some(t => t === VALIDATION.TYPES.METADATA);
-        if (isMetadata) {
+        if (isMetadata ) {
+            const docCount = await getCount(this.dataRecordsCollection, submissionID);
+            if (docCount === 0)  return ValidationHandler.handle([ERRORS.NO_VALIDATION_METADATA]);
+            else {
+                if (scope.toLowerCase() === VALIDATION.SCOPE.NEW ){
+                    const newDocCount = await getCount(this.dataRecordsCollection, submissionID, scope);
+                    if (newDocCount == 0)
+                        return ValidationHandler.success();
```

Comment:
Will this cause the metadataValidationStatus being stuck in "Validating" status? It seems it will, since no message sent to validation service, so no one will update metadataValidationStatus?